scripts/pyro_name_server.py

```diff
@@ -14,6 +14,6 @@
     parser.add_argument("--host", help="hostname or IP address to bind the server on")
     parser.add_argument("--port", help="port number to bind the server on")
     parser.add_argument("--autoclean", help="interval between registration autoclean (0=disabled)",
```

Comment:
I'm unclear on what "registration autoclean" means.

Reply:
Pyro objects register with the name server to allow their URI to be looked up by name, metadata matches, etc. The name server has an option to automatically remove the registrations of objects that it is unable to contact, useful for server crashes, network issues, etc. This is the 'autoclean' function, and to enable it you specify an interval between cleanings.